functional_tests/configuration_switching/configuration_switching_test.go

```diff
@@ -236,6 +245,7 @@ func testClusterReceiverEnabledOrDisabled(t *testing.T) {
 			"LogObjectsHecEndpoint":  logsObjectsHecEndpoint,
 		}
 		deployChartsAndApps(t, valuesFileName, replacements)
+		internal.WaitForTerminatingPods(t, clientset, internal.Namespace)
```

Comment:
Is this step for waiting on collector pod? Just wondering which pod could be terminating after the collector is installed (or is this an upgrade?) in the deployChartsAndApps.

Reply:
Yes, correct.
There was `ResetLogsSink` at the beginning but the better place for this is just after `WaitForTerminatingPods` - I moved it there
The second test has this (`ResetLogsSink`) in correct place